scene/2d/physics/collision_shape_2d.cpp

```diff
@@ -232,7 +227,18 @@ real_t CollisionShape2D::get_one_way_collision_margin() const {
 	return one_way_collision_margin;
 }
 
+#ifdef DEBUG_ENABLED
+
+Color CollisionShape2D::_get_default_debug_color() const {
+	SceneTree *st = SceneTree::get_singleton();
+	return st ? st->get_debug_collisions_color() : Color(0.0, 0.0, 0.0, 0.0);
```

Comment:
Wouldn't it be better to keep `Color()` (black) instead of transparent black as the fallback?

If using collision shapes without a scene tree, a black shape is probably better than a transparent one.

Reply:
This is debatable. On the one hand, at least you'll see the shape. On the other hand, you might get a bunch of indistinguishable black rectangles, squares, spheres, etc. But in any case, this is a very hypothetical scenario, the vast majority of users use the standard `SceneTree`, and do not implement their own `MainLoop`s.

Reply:
So would you prefer a transparent color?

Reply:
Why can't we use a fallback value that's actually a good option, like the current default?
```
1837:   debug_collisions_color = GLOBAL_DEF("debug/shapes/collision/shape_color", Color(0.0, 0.6, 0.7, 0.42));
```

Reply:
I would be for a better default too, although having skimmed over some code related to it, I think it's been used as a special value to mean "null". Maybe that should be checked as well